src/renderer/marketplace/bank-accounts/list/offers-table.jsx

```diff
@@ -133,24 +134,25 @@ export const BankingOffersTable = withStyles(styles)(
 					{data.map(bank => (
 						<TableRow key={bank.id}>
 							<TableCell className={classes.flagCell}>
-								<FlagCountryName code={bank.countryCode} size="small" />
+								<FlagCountryName code={bank['Country Code']} size="small" />
```

Comment:
This should be handled by a selector
- getBankPirce
- marchine readable fields like `bank['Min Deposit']` -> `bank.minDeposit`

Reply:
Agree, component should expect machine readable fields.